### PR TITLE
docs: Add guc verify_gpfdists_cert

### PR DESCRIPTION
**Pull request by mkiyama**
-added guc definition to list of gucs
-added link to guc from appropriate topics.

PR for 5X_STABLE
Will be ported to MAIN

See PR https://github.com/greenplum-db/gpdb/pull/4835

### CODE REVIEW

**Comment by mkiyama** [comment]
@xiong-gang Please review the doc updates for the verify_gpfdists_cert GUC

thanks

File: gpdb-doc/dita/admin_guide/external/g-gpfdists-protocol.xml [hunk 1]
```diff
@@ -50,5 +50,14 @@
     </ul>
     <p>For an example of loading data into an external table security, see <xref
         href="g-example-3-multiple-gpfdists-instances.xml#topic47"/>.</p>
+    <p>The server configuration parameter <codeph><xref
+          href="../../ref_guide/config_params/guc-list.xml#verify_gpfdists_cert"
+          >verify_gpfdists_cert</xref></codeph> controls whether SSL certificate authentication is
+      enabled when the external table communicates with the <codeph>gpfdist</codeph> utility to
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/external/g-gpfdists-protocol.xml]: It seems a little strange to say "the external table communicates" with gpfdist.  Maybe just change this to "Greenplum Database communicates"?

File: gpdb-doc/dita/admin_guide/external/g-gpfdists-protocol.xml [hunk 2]
```diff
@@ -50,5 +50,14 @@
...
+      either read data from or write data to an external data source. You can set the this parameter
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/external/g-gpfdists-protocol.xml]: Typo:  "the this"

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 3]
```diff
@@ -9874,6 +9875,48 @@
       </table>
     </body>
   </topic>
+  <topic id="verify_gpfdists_cert">
+    <title>verify_gpfdists_cert</title>
+    <body>
+      <p dir="ltr">When a Greenplum Database external table is defined with the
+          <codeph>gpfdists</codeph> protocol to use SSL security, this parameter controls whether
+        SSL certificate authentication is enabled. The default is <codeph>true</codeph>, SSL
+        authentication is enabled when the external table communicates with the
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: Same edit here, maybe.

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 4]
```diff
@@ -9874,6 +9875,48 @@
...
+          <codeph>gpfdist</codeph> utility to either read data from or write data to an external
+        data source.</p>
+      <p>The value <codeph>false</codeph> disables SSL certificate authentication. You can set the
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: typo: the this

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 5]
```diff
@@ -9874,6 +9875,48 @@
...
+        this parameter to <codeph>false</codeph> to disable authentication when testing the
+        communication between the Greenplum Database external table and the <codeph>gpfdist</codeph>
+        utility that is serving the external data.</p>
+      <note type="warning">Disabling SSL certificate authentication exposes a security risk by not
+        validating the <codeph>gpfdists</codeph> SSL certificate. </note>
+      <p>For information about <codeph>gpfdists</codeph> protocol, see <xref
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: about -> about the

**Comment by xiong-gang** [comment]
LGTM
@lij55 Please help review

**Comment by lij55** [comment]
Suggest to describe the behavior a little more clear:

If this GUC is set to false, following exception will be ignored:
* self-signed certificate used by gpfdist that is not trusted by Greenplum
* host name contained in certificate doesn't match the host name that gpfdist is run on

**Comment by mkiyama** [comment]
Thanks @lij55  @dyozie 
updated docs based on review comments.

File: gpdb-doc/dita/admin_guide/external/g-gpfdists-protocol.xml [hunk 6]
```diff
@@ -53,11 +53,17 @@
...
-      Greenplum Database external table and the <codeph>gpfdist</codeph> utility that is serving the
-      external data.<note type="warning">Disabling SSL certificate authentication exposes a security
-        risk by not validating the <codeph>gpfdists</codeph> SSL certificate. </note></p>
+      enabled when Greenplum Database communicates with the <codeph>gpfdist</codeph> utility to
+      either read data from or write data to an external data source. You can set the parameter
+      value to <codeph>false</codeph> to disable authentication when testing the communication
+      between the Greenplum Database external table and the <codeph>gpfdist</codeph> utility that is
+      serving the external data. If the value is <codeph>false</codeph>, these SSL exceptions are
+        ignored:<ul id="ul_vvb_5lj_mdb">
+        <li>The self-signed SSL certificate that is used by <codeph>gpfdist</codeph> that is not
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/external/g-gpfdists-protocol.xml]: Remove the second "that" in this sentence.  Also add a period to match second bullet item.

File: gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml [hunk 7]
```diff
@@ -185,24 +185,33 @@
...
+      <p>The server configuration parameter <codeph><xref
+            href="../../ref_guide/config_params/guc-list.xml#verify_gpfdists_cert"
+            >verify_gpfdists_cert</xref></codeph> controls whether SSL certificate authentication is
+        enabled when Greenplum Database communicates with the <codeph>gpfdist</codeph> utility to
+        either read data from or write data to an external data source. You can set the parameter
+        value to <codeph>false</codeph> to disable authentication when testing the communication
+        between the Greenplum Database external table and the <codeph>gpfdist</codeph> utility that
+        is serving the external data. If the value is <codeph>false</codeph>, these SSL exceptions
+        are ignored:<ul id="ul_vvb_5lj_mdb">
+          <li>The self-signed SSL certificate that is used by <codeph>gpfdist</codeph> that is not
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml]: Same edit here.

**Pull request merged**